pex/pex_info.py

```diff
@@ -51,8 +56,6 @@ class PexInfo(object):
 
     @classmethod
     def make_build_properties(cls, interpreter=None):
-        from .interpreter import PythonInterpreter
```

Comment:
This import may have been intentionally lazy (although it's not written down). I might recommend keeping it in this block, although making it absolute instead of relative.

Reply:
I was looking if the lazy import would actually end up mattering in production. It looks like no. It doesn't impact the imports of this file; if you only ever import `pex_info.py`, then you end up transitively importing way more than before.

But the only two places we import this file in production already import a ton of other things, `resolver.py` and `pex/pex.py`. So this shouldn't matter.

Reply:
In general, I would normally say if there's no comment, feel free to kill it, but I'm concerned about performance regressions since @jsirois isn't around. If he could make sure of that when he gets back that would be great, but otherwise we can 🚢 for now!